templates/tfengine/components/cicd/manual/README.md

```diff
@@ -24,14 +24,19 @@ to detect changes in the repo, trigger builds and run the workloads.
     * `TRIGGER_ENABLED`: Whether or not to enable all Cloud Build Triggers
```

Comment:
I think the README should stay under just cicd/, that seems like where users would expect to find it.

Reply:
This README.md is meant to show in the generated cicd/ directory at root. I added a separate one to describe the component itself.